pkg/tenant/api/v1/client.go

```diff
@@ -101,6 +101,32 @@ func (s *APIv1) SignUp(ctx context.Context, in *ContactInfo) (err error) {
 	return nil
 }
 
+func (s *APIv1) Register(ctx context.Context, in *RegisterRequest) (out *RegisterReply, err error) {
+	var req *http.Request
+	if req, err = s.NewRequest(ctx, http.MethodPost, "/v1/register", in, nil); err != nil {
+		return nil, err
+	}
+
+	out = &RegisterReply{}
+	if _, err = s.Do(req, out, true); err != nil {
+		return nil, err
+	}
+	return out, nil
+}
+
+func (s *APIv1) Login(ctx context.Context, in *LoginRequest) (out *AuthReply, err error) {
```

Comment:
Let's create a story for also adding `Refresh` here which Prema implemented today; sorry I forgot to add it to this story, but I don't want to block the completion of this PR.